en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/concepts/apim-as-control-plane.md

```diff
@@ -1,173 +1,25 @@
 # Choreo Connect with API Manager as Control Plane
 
-Choreo Connect can connect to API Manager running on cloud or on-premise as its control plane. Choreo Connect can be configured to connect with
-APIM control plane, therefore the user actions like API deploying, application creation, key generation, subscription creation etc are received by the Choreo Connect seamlessly.
+## Overview
 
-[![Choreo Connect Overview]({{base_path}}/assets/img/deploy/mgw/mgw_overview.png){: style="width:80%"}]({{base_path}}/assets/img/deploy/mgw/mgw_overview.png)
+Choreo Connect can connect to WSO2 API Manager (WSO2 API-M), which is running on cloud or on-premise, as its Control Plane. You can configure Choreo Connect to connect with API-M as the Control Plane, so that the user actions, such as API deploying, application creation, key generation, subscription creation, etc. are received by Choreo Connect seamlessly.
```

Comment:
```suggestion
Choreo Connect can connect to WSO2 API Manager (WSO2 API-M), which is running on cloud or on-premise. You can configure Choreo Connect to connect with API-M as the Control Plane, so that the user actions, such as API deploying, application creation, key generation, subscription creation, etc. are received by Choreo Connect seamlessly.
```